grype/db/curator.go

```diff
@@ -148,13 +152,14 @@ func (c *Curator) Update() (bool, error) {
 	defer downloadProgress.SetCompleted()
 	defer importProgress.SetCompleted()
 
-	updateAvailable, metadata, updateEntry, err := c.IsUpdateAvailable()
+	metadata := c.GetMetadata()
```

Comment:
Should we always download the listing on `grype db check` and `grype db update`?

Reply:
I don't think so, if we reliably know they are already up-to-date, why waste CPU cycles/network?

Reply:
I am pessimistically assuming that something will go wrong, and having a command that means, "no really, I mean it, download that database whether you need to or not" might be helpful.

The goal of this change is to alleviate traffic. Making rare commands less useful isn't going to alleviate traffic.

But I do see your point.

Reply:
I'm not sure how this is making commands less useful, they all continue to function exactly as they do: if we downloaded a listing file and got the latest database, we have the latest database, so downloading a listing again and downloading the database again is rather pointless. I think I understand your point, though. I'll see about making this particular function isolated as much as possible.

Reply:
I think as a user, I would prefer `grype db update` to be efficient and use this function to eliminate unnecessary downloads if it's already up-to-date. Maybe we add a `--force` argument or similar to make it download everything anew?